.github/CODEOWNERS

```diff
@@ -0,0 +1,15 @@
+# Based off the GitHub CODEOWNERS template at https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#example-of-a-codeowners-file
+
+################################################################################
+# These owners will be the default owners for everything in the repo. Unless a
+# later match takes precedence, @global-owner1 and @global-owner2 will be
+# requested for review when someone opens a pull request.
+*       @matt-bernhardt @mitlibraries/engx
```

Comment:
Probably.  I'm still working through some of my thoughts on how we want to make use of the `CODEOWNERS` file to share/track information about the repo. I think that it would be sufficient to just have the `@mitlibraries/engx` here, but keeping `@matt-bernhardt` here helps indicate that you are the primary owner of this repository. I believe (but could maybe be convinced otherwise) that there should always be at least one individual who is flagged as an owner (instead of just a group as an owner) to avoid "well, someone else in my group will deal with this" problem when the primary owner is a group.

Reply:
I can see that argument, yeah. Ultimately I think this is something that should be decided across DLS and between team leads like yourself and Jeremy, but in terms of this PR I'm fine with it being this way at this time.